components/server/src/server.ts

```diff
@@ -383,7 +383,7 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
             });
         }
 
-        this.apiServer = this.api.listen(9877);
```

Comment:
was that removal accidental?

Reply:
no, it is not used

Reply:
Ah, I intended to receive the port in the arg, but then forgot to actually propagate the port from the caller. Thanks for the cleanup here.